src/views/resultsView/astDetailsView.ts

```diff
@@ -258,45 +303,61 @@ export class AstDetailsDetached implements vscode.WebviewViewProvider {
         </head>
         <div id="main_div">
           ${this.result.type !== "sca" ? html.header(severityPath) : ""}
-          ${this.result.type === "sast"
-        ? html.tab(
-          html.generalTab(cxPath),
-          html.detailsTab(),
-          html.changes(selectClassname),
-          messages.generalTab,
-          messages.descriptionTab,
-          messages.triageTab,
-          messages.remediationExamplesTab,
-          messages.noRemediationExamplesTab,
-          isAIEnabled ? `${constants.aiSecurityChampion}` : "",
-          isAIEnabled ? html.guidedRemediationSastTab(cxIcon, masked) : ""
-        )
-        : this.result.type === "sca"
-          ? html.scaView(
-            severityPath,
-            scaAtackVector,
-            scaComplexity,
-            scaAuthentication,
-            scaConfidentiality,
-            scaIntegrity,
-            scaAvailability,
-            scaUpgrade,
-            scaUrl,
-            this.type
-          )
-          : html.tab(
-            html.generalTab(cxPath),
-            "",
-            html.changes(selectClassname),
-            messages.generalTab,
-            "",
-            messages.triageTab,
-            "",
-            "",
-            isAIEnabled ? `${constants.aiSecurityChampion}` : "",
-            isAIEnabled ? html.guidedRemediationTab(kicsIcon, masked) : ""
-          )
-      }
+          ${
+            this.result.type === "sast"
+              ? html.tab(
+                  html.generalTab(cxPath),
+                  html.detailsTab(),
+                  html.changes(selectClassname),
+                  messages.generalTab,
+                  messages.descriptionTab,
+                  messages.triageTab,
+                  messages.remediationExamplesTab,
+                  messages.noRemediationExamplesTab,
+                  isAIEnabled ? `${constants.aiSecurityChampion}` : "",
+                  isAIEnabled
+                    ? html.guidedRemediationSastTab(cxIcon, masked)
+                    : ""
+                )
+              : this.result.type === "sca"
+              ? html.scaView(
+                  severityPath,
+                  scaAtackVector,
+                  scaComplexity,
+                  scaAuthentication,
+                  scaConfidentiality,
+                  scaIntegrity,
+                  scaAvailability,
+                  scaUpgrade,
+                  scaUrl,
+                  this.type
+                )
+              : this.result.type === constants.scs
+              ? html.tab(
+                  html.generalTab(cxPath),
+                  html.scsDetailsDescriptionTab(),
+                  html.scsDetailsRemediationTab(),
+                  messages.generalTab,
+                  messages.descriptionTab,
+                  messages.remediationExamplesTab,
+                  "",
+                  "",
+                  "",
+                  ""
+                )
+              : html.tab(
```

Comment:
this else is not good, add else if result.type == kicstype

if its non of out supported engines type, we should ignore the result

Reply:
Im not sure we must have this, because until now the else existed and handle the "kicks" case.
Mabye there is a reason for such general condtion.

Reply:
But if the results will contain by mistake unsupported engine, this "else" will parse this unsupported engine as it parse kics